core/src/main/java/com/onelogin/saml2/http/HttpResponse.java

```diff
@@ -0,0 +1,14 @@
+package com.onelogin.saml2.http;
+
+import java.io.IOException;
+
+/**
+ * Framework-agnostic representation of an HTTP response.
+ *
+ * @since 3.0.0
+ */
+public interface HttpResponse {
```

Comment:
I'm OK calling this interface `SamlHttpResponse` or `OneLoginSamlHttpResponse` if the existing name is too generic.